signer/tests/integration/postgres.rs

```diff
@@ -1446,27 +1455,23 @@ async fn get_swept_deposit_requests_returns_swept_deposit_requests() {
 
     // Its details should match that of the deposit request.
     let req = requests.pop().unwrap();
+
     assert_eq!(req.amount, setup.deposit_request.amount);
     assert_eq!(req.txid, setup.deposit_request.outpoint.txid.into());
     assert_eq!(req.output_index, setup.deposit_request.outpoint.vout);
     assert_eq!(req.recipient, setup.deposit_recipient.into());
     assert_eq!(req.sweep_block_hash, setup.sweep_block_hash.into());
     assert_eq!(req.sweep_block_height, setup.sweep_block_height);
     assert_eq!(req.sweep_txid, setup.sweep_tx_info.txid.into());
-    assert_eq!(req.sweep_tx, setup.sweep_tx_info.tx.into());
+    //assert_eq!(swept_deposit.sweep_tx, setup.sweep_tx_info.tx.into());
```

Comment:
We can remove this line right? Or at least uncomment it.

Reply:
Yeah, removed in [21fab00](https://github.com/stacks-network/sbtc/pull/710/commits/21fab009a0ddf33f79b4b9928e06d67073656e5d). It was commented out because the raw tx is no longer a field on the swept deposit.